rest/parse.go

```diff
@@ -0,0 +1,33 @@
+package rest
```

Comment:
move this to util package

Reply:
Even though this is specific to URL parsing?